network/endpoint_windows.go

```diff
@@ -529,6 +549,13 @@ func (nw *network) deleteEndpointImpl(_ netlink.NetlinkInterface, _ platform.Exe
 		return nil
 	}
 
+	if ep.NICType == cns.ApipaNIC {
+		if err := nw.deleteHostNCApipaEndpoint(ep.NetworkContainerID); err != nil {
+			logger.Error("Failed to delete HostNCApipaEndpoint due to error", zap.Error(err))
```

Comment:
nit: we log the error here, but don't we also return it and log it at the top level already?